docusaurus/docs/Android/03-ui/05-components/05-message-list.mdx

```diff
@@ -461,4 +461,19 @@ If your predicate doesn't show avatars for your own messages (this is the defaul
 
 ```xml
 streamUiMessageEndMargin="0dp"
-```
\ No newline at end of file
+```
+## Configure giphy
+It is possible to configure the gifs size. Prior of version 4.23.0 gifs are autosizable and will resize thenselves accordingly with the size of the image they load. It is possible to use constant size gifs using `streamUiMediaAttachmentConstantSize` and set the desired height using `streamUiMediaAttachmentGiphyHeight`.
+
+As of v5.x.x, our Giphy holders are much more customizable. You can use three Giphy load modes:
```

Comment:
Think it's okay saying as of v5.0.0 since we're talking about a starting point.
"As of 5.x.x" would imply that any 5.x.x. release was the starting point.